homeassistant/components/light/hyperion.py

```diff
@@ -19,7 +19,8 @@ def setup_platform(hass, config, add_devices_callback, discovery_info=None):
     """Setup a Hyperion server remote."""
     host = config.get(CONF_HOST, None)
     port = config.get("port", 19444)
-    device = Hyperion(config.get('name', host), host, port)
+    color = config.get("color", [255, 255, 255])
```

Comment:
What's with the color? How is that related to show whether the light is on or off?